crates/llm-local/src/bert.rs

```diff
@@ -4,7 +4,7 @@
 ///
 /// TODO: Remove this file when a new release of Candle makes it obsolete.
```

Comment:
Has this still not happened?

Reply:
I would prefer to leave the embeddings part unchanged in this PR if that is acceptable.